### ISSUE TITLE
Blade format nor working

### ISSUE THREAD

**Issue by fidalgodev** [onecentlin/laravel-blade-snippets-vscode#74]
Installed the plugin, also running the latest version of VS code.

Im on a blade.php file, on the bottom right there is the "Laravel Blade" file type. But when i type b: nothing shows up besides Emmet Abreviation "<b:></b:>", and when i press tab it triggers "<b:></b:>".

Also tried b:extends, and all the others, and nothing. 

My preferences file:

```
{
    "git.ignoreMissingGitWarning": true,
    "workbench.iconTheme": "vscode-icons",
    "explorer.confirmDelete": false,
    "terminal.integrated.shell.windows": "C:\\Program Files\\Git\\bin\\bash.exe",
    "workbench.colorTheme": "One Dark Pro",
    "emmet.triggerExpansionOnTab": true, // enable tab to expanse emmet tags
    "blade.format.enable": true,         // if you would like to enable blade format

}
```

If i type only "b", i get all emmet suggestions, and then i scroll down and i get all Blade Snippet suggestions.
But as soon as i type "b:" every suggestion goes away and im only left with Emmet Abreviation "<b:></b:>". How can i fix this?

**Comment by pedrofidalgomrs** [comment]
If i type only "b", i get all emmet suggestions, and then i scroll down and i get all Blade Snippet suggestions.
But as soon as i type "b:" every suggestion goes away and im only left with  Emmet Abreviation "<b:></b:>". How can i fix this?

**Comment by Nernofire** [comment]
The exact same issue with me. VS runing on windows 7 x64

**Comment by arifinofficial** [comment]
Problem same with me on VSCode version 1.25

**Comment by sloughrey** [comment]
Same issue here! 

Win 10 x64 
VSCode version: 1.25.1

**Comment by sagarpanchal** [comment]
Version: 1.25.1
Commit: 1dfc5e557209371715f655691b1235b6b26a06be
Date: 2018-07-11T15:40:20.190Z
Electron: 1.7.12
Chrome: 58.0.3029.110
Node.js: 7.9.0
V8: 5.8.283.38
Architecture: x64

**Comment by sagarpanchal** [comment]
Same issue

```
Version: 1.25.1
Commit: 1dfc5e557209371715f655691b1235b6b26a06be
Date: 2018-07-11T15:40:20.190Z
Electron: 1.7.12
Chrome: 58.0.3029.110
Node.js: 7.9.0
V8: 5.8.283.38
Architecture: x64
```

**Comment by Zaen8** [comment]
Same issue here, for some reason I bypassed it typing the abreviation without the colon. As @pedrofidalgomrs points out the problem appears after using the colon.

For example, `bextends` and pressing tab works for me.

**Comment by sagarpanchal** [comment]
@Zaen8 that's because you're using [laravel-easy-blade-snippets](https://github.com/dacoto-com/laravel-easy-blade-snippets)

**Comment by pedrofidalgomrs** [comment]
Im not using that. Using Laravel Blade Snippets

**Comment by Zaen8** [comment]
@sagarpanchal I'm not using that extension either.
![imagen](https://user-images.githubusercontent.com/29967529/43955740-d011f640-9ca1-11e8-8151-8760c6d7b8b8.png)
@pedrofidalgomrs Does your snippets work when you press tab without using the colon?

**Comment by petem46** [comment]
I was having this issue.  Resolved by ensuring I only had the Laravel extensions shown in screenshot above from @Zaen8 
Using the colom breaks it - 
• bextends => @extends('name')
• bext => @extends('name')
• b:extends => <b:extends></b:extends>
• b:ext => <b:ext></b:ext>

**Comment by pedrofidalgomrs** [comment]
Yeah, without using the ":" it works fine.. Just use bext and press tab
